### PR TITLE
update for typography and links to more info

### PR DESCRIPTION
**Pull request by mhewson**

### CODE REVIEW

File: docs/release-notes/fall-2019/README.md [hunk 1]
```diff
@@ -45,7 +45,11 @@ TODO: how should components vs. micro-sites handle icons with the new repo?
 
 ### New Font/Typography
 
-TODO: ????
+Sentinal and Roboto have been deprecated as global REI brand fonts. cdr-fonts.css has removed the base64 encoded versions of sentinal fonts as a result and has greatly reducing the overall size of this file.
```
Comment (cowills) [inline docs/release-notes/fall-2019/README.md]: sentinel?

Comment (cowills) [reply docs/release-notes/fall-2019/README.md]: "reduced

File: docs/release-notes/fall-2019/README.md [hunk 2]
```diff
@@ -45,7 +45,11 @@ TODO: how should components vs. micro-sites handle icons with the new repo?
 
 ### New Font/Typography
 
-TODO: ????
+Sentinal and Roboto have been deprecated as global REI brand fonts. cdr-fonts.css has removed the base64 encoded versions of sentinal fonts as a result and has greatly reducing the overall size of this file. 
+
+the variant fonts for "REI Stuart App" and "Graphik App" have been added.
+
+[more infomation on typeography](../../foundation/typography/?active-tab=guidelines)
```
Comment (cowills) [inline docs/release-notes/fall-2019/README.md]: typeo

Comment (mhewson) [reply docs/release-notes/fall-2019/README.md]: lord above

**Pull request merged**